internal/querycoord/task_scheduler.go

```diff
@@ -361,6 +361,13 @@ func (scheduler *TaskScheduler) unmarshalTask(taskID UniqueID, t string) (task,
 		if err != nil {
 			return nil, err
 		}
+		// in case after update milvus, can't find recover watch type
+		// different queryNode may watch same dmChannel and create same growing segment
+		// deduplicate result when reduce, the correctness of search has no effect
+		// and growing segment will be removed after handoff
+		if loadReq.WatchType == querypb.WatchType_WatchTypeNone {
```

Comment:
This is weird, do we has some meta about this collection whether it is loaded per collection or per segment?

Reply:
also I guess delta channel need to be handled at the same time